languages/elixir/reference/README.md

```diff
@@ -27,36 +27,49 @@ Below are the concepts that have been identified in Elixir for use in Exercism.
 - elixir mix
   - directory structure
   - mix tasks
+    - format
+    - clean
+    - test
 - Naming conventions
 - Eager Computation
 - Lazy Computation
 - Guards
 - Compiling
 - Scripts (*.exs) vs Code (*.ex)
+- Regex
 - TODO: more
 
 ### Modules
 
 - Naming
 - Attributes
+  - Redefining
 - Alias
 - Require
 - Import
 - Use
+- Dynamic
```

Comment:
What does dynamic mean here regarding modules? The only thing that really comes to mind for me is dependency injection of the module, but I'm not sure that's really a concept unto itself.